tests/study_tests/test_optimize.py

```diff
@@ -58,7 +57,7 @@ def func_nan(_: Trial) -> float:
 # TODO(Yanase): Remove this test function after removing `optuna.structs.TrialPruned`.
```

Comment:
I'm wondering if we can remove this test as @toshihikoyanase said here, as well. cc. @toshihikoyanase

Reply:
Thanks! I deleted this test